web/src/modules/topic/components/TopicToolbar/TopicToolbar.tsx

```diff
@@ -59,10 +59,10 @@ export const TopicToolbar = () => {
           <MenuItem onClick={() => loadExample("world_hunger.json")}>World Hunger</MenuItem>
           <MenuItem onClick={() => loadExample("ameliorate.json")}>Ameliorate</MenuItem>
         </Menu>
-        <IconButton color="inherit" onClick={downloadTopic}>
+        <IconButton color="inherit" title="Download" aria-label="Download" onClick={downloadTopic}>
```

Comment:
nice usage of `title` - I was thinking a tooltip component would have to be used but this seems to do the trick nicely.

Was trying to figure out if `aria-label` is necessary if `title` is set... TIL about devtools' [accessibility tab](https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/accessibility/accessibility-tab), which actually shows that the Name property is being set without aria-label. However, a few different sources ([random blog post](https://inclusive-components.design/tooltips-toggletips/), [stack overflow](https://stackoverflow.com/a/63726152)) critique sole usage of `title`, so I think it's good to set `aria-label` as well to be safe.